tests/test_redirects.py

```diff
@@ -0,0 +1,74 @@
+def setup_app(app):
+    from lona import LonaView
+
+    # root
+    @app.route('/')
+    class RootView(LonaView):
+        def handle_request(self, request):
+            return 'ROOT'
+
+    @app.route('/redirect-to-root/')
+    class RedirectToRoot(LonaView):
+        def handle_request(self, request):
+            return {
+                'redirect': '/',
+            }
+
+    # absolute url
+    @app.route('/foo/')
+    class FooView(LonaView):
+        def handle_request(self, request):
+            return 'FOO'
+
+    @app.route('/redirect-to-foo/')
+    class RedirectToFooView(LonaView):
+        def handle_request(self, request):
+            return {
+                'redirect': '/foo/',
+            }
+
+    # relative url
+    relative_redirected = False
+
+    @app.route('/bar/')
+    class BarFooView(LonaView):
+        def handle_request(self, request):
+            nonlocal relative_redirected
+
+            if not relative_redirected:
+                relative_redirected = True
+
+                return {
+                    'redirect': '.',
+                }
+
+            else:
+                return 'BAR'
+
+
+async def test_rendering(lona_app_context):
+    """
+    This test tests redirects by creating multiple views and redirecting from
+    one to another. Absolute and relative URLs are tested.
+    """
+
+    from playwright.async_api import async_playwright
+
+    context = await lona_app_context(setup_app)
+
+    async with async_playwright() as p:
+        browser = await p.chromium.launch()
+        browser_context = await browser.new_context()
+        page = await browser_context.new_page()
+
+        # test redirect to root
+        await page.goto(context.make_url('/redirect-to-root/'))
```

Comment:
Let's assert url as well
```suggestion
        await page.goto(context.make_url('/redirect-to-root/'))
        await page.wait_for_url('/')
```